pilot/pkg/model/virtualservice.go

```diff
@@ -601,10 +601,6 @@ func VirtualServiceDependencies(vs config.Config) []ConfigKey {
 			// shouldn't happen
 			continue
 		}
-		if len(knn) < 2 {
```

Comment:
I think it is useful to prevent panic in L608

Reply:
should i create a new pr after this merge?